src/tests/blueprint/t_blueprint_mesh_verify.cpp

```diff
@@ -0,0 +1,1163 @@
+}
+
+
+//-----------------------------------------------------------------------------
+TEST(conduit_blueprint_mesh_verify, topology_general)
+{
+    VerifyFun verify_topology_funs[] = {
+        blueprint::mesh::topology::verify,
+        verify_topology_protocol};
+
+    for(index_t fi = 0; fi < 2; fi++)
+    {
+        VerifyFun verify_topology = verify_topology_funs[fi];
+
+        Node mesh, info;
+        EXPECT_FALSE(verify_topology(mesh,info));
+
+        blueprint::mesh::examples::braid("quads",10,10,1,mesh);
+        Node& n = mesh["topologies"]["mesh"];
+
+        { // Type Field Tests //
+            n.remove("type");
+            EXPECT_FALSE(verify_topology(n,info));
+            n["type"].set("explicit");
+            EXPECT_FALSE(verify_topology(n,info));
+
+            // FIXME: Remove the comments from the following line once the verify functions
+            // for uniform and rectilinear topologies have been implemented.
+            const std::string topology_types[] = {/*"uniform", "rectilinear", */"structured"};
+            for(index_t ti = 0; ti < 1; ti++)
+            {
+                n["type"].set(topology_types[ti]);
+                EXPECT_FALSE(verify_topology(n,info));
+            }
+
+            n["type"].set("unstructured");
+            EXPECT_TRUE(verify_topology(n,info));
+        }
+
+        { // Coordset Field Tests //
+            n.remove("coordset");
+            EXPECT_FALSE(verify_topology(n,info));
+
+            n["coordset"].set(0);
+            EXPECT_FALSE(verify_topology(n,info));
+
+            n["coordset"].set("coords");
+            EXPECT_TRUE(verify_topology(n,info));
+        }
+
+        { // Grid Function Field Tests //
+            n["grid_function"].set(10);
+            EXPECT_FALSE(verify_topology(n,info));
+            n["grid_function"].set("coords_gf");
+            EXPECT_TRUE(verify_topology(n,info));
+        }
+    }
+}
+
+/// Mesh Field Tests ///
+
+//-----------------------------------------------------------------------------
+TEST(conduit_blueprint_mesh_verify, field_association)
+{
+    Node n, info;
+    EXPECT_FALSE(blueprint::mesh::field::association::verify(n,info));
+
+    const std::string shape_types[] = {"point", "element"};
+    for(index_t ti = 0; ti < 2; ti++)
+    {
+        n.reset();
+        n.set(shape_types[ti]);
+        EXPECT_TRUE(blueprint::mesh::field::association::verify(n,info));
+    }
+
+    n.reset();
+    n.set(0);
+    EXPECT_FALSE(blueprint::mesh::field::association::verify(n,info));
+
+    n.reset();
+    n.set("zone");
+    EXPECT_FALSE(blueprint::mesh::field::association::verify(n,info));
+}
+
+
+//-----------------------------------------------------------------------------
+TEST(conduit_blueprint_mesh_verify, field_basis)
+{
+    // FIXME: Does this have to be verified against anything else?  What does
+    // this basis refer to if it isn't a different path in the mesh structure?
+    Node n, info;
```

Comment:
Thanks for the clarification on this.  I guess it isn't too important that we verify it for now, but I think that keeping the 'fixme' here will be helpful to remind us to expand on this test if/when we decide to expand the `blueprint::mesh::field::basis::verify` function.